### ISSUE TITLE
Separate feasts by sanctorale/temporale

### ISSUE THREAD

**Issue by blueNoteb5** [DDMAL/CantusDB#25]

**Comment by jwang44** [comment]
The separation is based on the prefix (first two digits) of the feast code. However, implementation on the old Cantus is different from the instructions given for the new Cantus. This will be put on hold for now.

**Comment by dchiller** [comment]
@annamorphism This is a long-ago issue...I'm trying to determine how much this is still an issue. 

Currently, we allow filtering by sanctorale or temporale on the [feast list page](https://cantusdatabase.org/feasts/). This is achieved by looking at the feast code. Feast codes that begin with one of these character sets are temporale: "01", "02", "03", "04", "05", "06","07", "08", "09", "10", "11", "16", "17". Feast codes that begin with one of these character sets are sanctorale: "12", "13", "14", "15".

Can we consider this separation good enough? I'm wondering if this seems like the right way to differentiate? Are these feast codes Cantus Network-specific?

**Comment by annamorphism** [comment]
Yes, these codes are Cantus Network specific, as far as I'm aware. 
I think the problem this was intended for at the time was just to have a dropdown exist, which we now have, so in that sense this is now solved.
I'm not really sure why 16 and 17 are Temporale (maybe there is documentation of the rationale somewhere...). 17 looks like Sanctorale for which the date is variable or unknown? And 16 and 13 are both kinds of rites without fixed dates...I'm going to look into this, but I feel like this is probably fine.

**Comment by dchiller** [comment]
Sounds good! The division of the various feast code prefixes are things we can change, so if in your digging some assignments seem wrong, we can solve that with this issue. 

I'll keep this open to make sure to circle back to that, but otherwise we can consider this fulfilled.
